pkgs/development/python-modules/xacro/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  setuptools-scm,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "xacro";
+  version = "2.0.13";
+  pyproject = true;
```

Comment:
hmmm im not sure why nix-init did that, xacro has no pyproject.toml, just a setup.py